blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ActionProxiesImpl.java

```diff
@@ -8,14 +11,18 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
 /**
  * @author Vivek Pandey
  */
 public class ActionProxiesImpl extends BlueActionProxy {
-
     private final Action action;
     private final Reachable parent;
     private static final Logger logger = LoggerFactory.getLogger(ActionProxiesImpl.class);
+    private static final ImmutableSet<String> BANNED_ACTIONS = ImmutableSet.of("org.jenkinsci.plugins.workflow.job.views.FlowGraphAction");
```

Comment:
@vivek @michaelneale @tfennelly moved banned actions to a immutable set in this class.